go.mod

```diff
@@ -270,3 +270,5 @@ replace (
 	github.com/sirupsen/logrus => github.com/gravitational/logrus v1.4.4-0.20210817004754-047e20245621
 	github.com/vulcand/predicate => github.com/gravitational/predicate v1.2.1
 )
+
+replace github.com/gravitational/kingpin => /Users/stevehuang/go/github.com/gravitational/kingpin
```

Comment:
ignore go.mod and go.sum for now. will make a patch on our kingpin and update this.